### ISSUE TITLE
seem to have broken readmarker ghosts

### ISSUE THREAD

**Issue by matrixbot** [vector-im/element-web#1313]
Created by @ richvdh:matrix.org.

warning.js:45 Warning: flattenChildren(...): Encountered two children with the same key, .1:$_readuptoghost_$145929209245178luQop=2matrix=1org. Child keys must be unique; when two children share a key, only the first child will be used

**Comment by ara4n** [comment]
I believe this is fixed by the PR

**Comment by ara4n** [comment]
@freelock just saw this recur on current develop with @kegsay's IndexedDB stuff.  It should also be in a rageshake.

```
rageshake.js:63 Warning: flattenChildren(...): Encountered two children with the same key, `1:$_readuptoghost_$14874453102710gzqix:matrix.freelock.com`. Child keys must be unique; when two children share a key, only the first child will be used.
    in ol (created by ScrollPanel)
    in div (created by ScrollPanel)
    in div (created by GeminiScrollbar)
    in div (created by GeminiScrollbar)
    in GeminiScrollbar (created by ScrollPanel)
    in ScrollPanel (created by MessagePanel)
    in MessagePanel (created by TimelinePanel)
    in TimelinePanel (created by RoomView)
    in div (created by RoomView)
    in RoomView (created by LoggedInView)
    in main (created by LoggedInView)
    in div (created by LoggedInView)
    in div (created by LoggedInView)
    in LoggedInView (created by MatrixChat)
    in MatrixChat
```

**Comment by richvdh** [comment]
It's been broken forever. Unrelated to indexeddb.

**Comment by Kegsay** [comment]
/me is getting annoyed with the number of issues he's being pinged on with the assumption that indexeddb stuff is to blame. >:(

**Comment by ara4n** [comment]
Previous duplicate children have been caused by duplicate events in the timeline, which definitely feels like the sort of thing that a bug with IndexedDB could cause.  Eitherway, just pinging to be on the safe side and get a possible cause ruled out - and vdH has already pointed out that it's unrelated.  I don't think this is reason to get annoyed.

Meanwhile, I believe all the other issues where I've pinged for IndexedDB stuff are genuine bugs with it (with the exception of #3260, which was mentioning it again to rule it out as a possible factor)...
